cypress/page/index.js

```diff
@@ -0,0 +1,7 @@
+export { MenuContentPage }from "./menu-content.page"
```

Comment:
Maybe this file should be called index.ts, and should be the only index file on this page folder